Angular InstantSearch/algolia-insights/src/app/app.component.ts

```diff
@@ -16,13 +16,7 @@ const searchClient = algoliasearch(
 export class AppComponent {
   config = {
     indexName: "demo_ecommerce",
-    searchClient
+    searchClient,
+    insightsClient: aa
```

Comment:
```suggestion
    insightsClient: window.aa
```

Unless TypeScript complains?

Reply:
typescript doesn't like it. It returns `Property 'aa' does not exist on type 'Window'.`
I can use `(window as any).aa` however.

Reply:
done here: fa014f0